addons/l10n_fr_certification/models/account.py

```diff
@@ -74,6 +74,9 @@ def write(self, vals):
                 # restrict the operation in case we are trying to write a forbidden field
                 if (move.state == "posted" and set(vals).intersection(MOVE_FIELDS)):
                     raise UserError(ERR_MSG % (self._name, ', '.join(MOVE_FIELDS)))
+                # restrict the operation in case we are trying to overwrite existing hash
+                if (move.l10n_fr_hash and vals.get('l10n_fr_hash')) or (move.l10n_fr_secure_sequence_number and vals.get('l10n_fr_secure_sequence_number')):
```

Comment:
Hi Quentin I think you should only test 
`
if move.l10n_fr_hash or move.l10n_fr_secure_sequence_number:
`
Indeed I I do a write with {'l10n_fr_hash': '', 'l10n_fr_secure_sequence_number': ''} then a write with the new value I will not have any raise error.
What do you think?